extensionsGallery.json

```diff
@@ -5080,15 +5080,76 @@
 					],
 					"statistics": [],
 					"flags": ""
-				}
+				},
+				{
+				"extensionId": "100",
```

Comment:
please update this to 101 to match the id in the insiders gallery.

Reply:
Oh 100 was so beautiful ). Done.